src/plugin/morphologyHDF5.cpp

```diff
@@ -283,14 +301,15 @@ int MorphologyHDF5::_readSections()
         std::vector<std::vector<int>> vec;
         vec.resize(dims[0] - 1);
         dataset.read(vec);
-        bool skipFirst = true;
         int firstSectionOffset = vec[1][0];
+        sections.reserve(sections.size() + vec.size() - 1);
+        bool skipFirst = true;
         for (auto p : vec) {
             if (skipFirst) {
                 skipFirst = false;
                 continue;
             }
-            sections.push_back({p[0] - firstSectionOffset, p[1] - 1});
+            sections.emplace_back(Property::Section::Type{p[0] - firstSectionOffset, p[1] - 1});
```

Comment:
isn't it "almost" same given the move semantics?

Reply:
Wouldn't it require some additional `std::move`? I like to be consistent and just use `emplace_back` for non-primitive types.

Reply:
`{}` passed to `push_back` was already an rvalue as far as I can tell.
Whatever though.